code401challenges/src/test/java/code401challenges/tree/TreeTest.java

```diff
@@ -0,0 +1,39 @@
+package code401challenges.tree;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class TreeTest {
+
+    Tree<String> tree;
+
+    @Before
+    public void setup() {
+        tree = new Tree<String>(new Node<String>("A",
+                new Node<String>("B", new Node<String>("D"), new Node<String>("E")),
+                new Node<String>("C", new Node<String>("F"), new Node<String>(null))));
```

Comment:
Since you've already declared your tree variable as a Tree<String> on line 10, you don't need to put String within the carats.